pkg/auth/handler/webapp/viewmodels/base.go

```diff
@@ -245,7 +246,8 @@ func (m *BaseViewModeler) ViewModel(r *http.Request, rw http.ResponseWriter) Bas
 			url, _ = m.StaticAssets.GeneratedStaticAssetURL(id)
 			return
 		},
-		DarkThemeEnabled: !m.AuthUI.DarkThemeDisabled,
+		DarkThemeEnabled:  !m.AuthUI.DarkThemeDisabled || webapp.IsInlinePreviewPageRequest(r),
+		LightThemeEnabled: !m.AuthUI.LightThemeDisabled,
```

Comment:
It should be `!m.AuthUI.LightThemeDisabled || webapp.IsInlinePreviewPageRequest(r)`